### ISSUE TITLE
Filter results by tag

### ISSUE THREAD

**Issue by stevennoto** [rocketmiles/aws-cct#7]
It would be nice to be able to filter or aggregate results by tag, for example by app or by qa/prod.

It's unclear to me whether the Cost Explorer (and the Go SDK) supports tag filters on `GetCostAndUsageRequest`. I don't see it as a dimension at https://docs.aws.amazon.com/aws-cost-management/latest/APIReference/API_GetDimensionValues.html

**Comment by rgoomar** [comment]
It's possible as part of the CostUsage input (https://docs.aws.amazon.com/sdk-for-go/api/service/costexplorer/#GetCostAndUsageInput) and defining a filter expression to include tags. See - https://docs.aws.amazon.com/sdk-for-go/api/service/costexplorer/#Expression

**Comment by stevennoto** [comment]
Cool! I have a basic version of tag filtering working locally. The hard parts will be providing a clean interface in the cli, and then dynamically constructing a tree of `costexplorer.Expression`'s holding an expression like "service=blah AND tag1=val1".

Any preference for this? For example I am not sure if the cli library supports it but I was pondering something like:

```
./aws-cct --service "Amazon Elastic Compute Cloud - Compute" --tag app=unicorn --tag env=prod
```

**Comment by rgoomar** [comment]
Released in [v1.3.0](https://github.com/rocketmiles/aws-cct/releases/tag/v1.3.0)
